script.js

```diff
@@ -55,11 +55,11 @@ var rollDice = function() {
 //make players
 var player1 = new player("#player1");
 var player2 = new player("player2");
-var player3 = new player('player3');
-var player4 = new player('player4');
-var ai1 = new player('AI1');
-var ai2 = new player('AI2');
-var ai3 = new player('AI3');
+var player3 = new player("player3");
+var player4 = new player("player4");
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

<!-- Comment by [SideCI](https://sideci.com) -->